modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -385,8 +385,10 @@
 	"removing": "Removing",
 	"renew": "Renew",
 	"renew-key": "Renew Key",
-	"renew-x-activation-keys": "Renew {0} Activations Key(s)",
-	"renew-x-key": "Renew {0} Key(s)",
+	"renew-x-activation-key": "Renew {0} Activations Key",
```

Comment:
one more change for this. we can remove the `x` and just make it "Renew Activation Key" since its only one, i dont think we need a number on it.

Reply:
i will make fix and resend since you left on vacation